bcycle-backend/Models/Entities/GroupTripCode.cs

```diff
@@ -0,0 +1,14 @@
+using System.ComponentModel.DataAnnotations;
+
+namespace bcycle_backend.Models.Entities
+{
+    public class GroupTripCode
```

Comment:
Why is this a separate entity? Why is it not used by `GroupTrip`?

Reply:
It is only stored  in database to track utilized codes so we wont draw the duplicate. We dont need the connection with `GroupTrip` for that. Like i wrote somewhere in the code its sufficient for our case.

Reply:
Wouldn’t it be easier to just search GroupTrip instead of introducing a new table?

Reply:
May be easier but also more expensive. Separating concerns is also good. Additional table isn't that much of a deal. In our case its whatever though. I'll see what i feel about it tomorrow :)

Reply:
Done